packages/typescript/internal/ts_project.bzl

```diff
@@ -113,10 +114,15 @@ def _ts_project_impl(ctx):
     else:
         json_outs = []
 
+    # diagnostics outputs from tsc
+    exit_code_output = ctx.actions.declare_file("%s.exit.code" % ctx.label.name)
+    diagnostics_output = ctx.actions.declare_file("%s.diagnostics" % ctx.label.name)
+    processed_diagnostics_output = ctx.actions.declare_file("%s.diagnostics_processed" % ctx.label.name)
+
     outputs = json_outs + ctx.outputs.js_outs + ctx.outputs.map_outs + ctx.outputs.typings_outs + ctx.outputs.typing_maps_outs
     if ctx.outputs.buildinfo_out:
         outputs.append(ctx.outputs.buildinfo_out)
-    runtime_outputs = depset(json_outs + ctx.outputs.js_outs + ctx.outputs.map_outs)
+    runtime_outputs = depset(json_outs + ctx.outputs.js_outs + ctx.outputs.map_outs + [processed_diagnostics_output])
```

Comment:
I'm not sure this is the right place to add this output, but it needed to cause the diagnostic processor to run as otherwise nothing is requesting the output of the processors action. However ts_project has its default outputs for JS and d.ts files etc